src/assets/_project/_blocks/scss-general/_qg-variables.scss

```diff
@@ -340,13 +343,18 @@ $qg-linkedin: #0077b5;
 
 $text-underline-offset: 5px;
 
-@mixin qg-button-outline-decoration ($margin: 2px) {
+@mixin qg-button-outline-decoration($margin: 2px, $light-background: true) {
```

Comment:
Wouldn't it make more sense to pass a colour than a boolean? It could default to `$qg-active-outline` and the calling code could pass in `$qg-active-outline-brighter` as needed.

Reply:
Good idea. Just updated the code.